### ISSUE TITLE
[A11y] Use `aria-current="page"` for the page menu in `create-svelte` template

### ISSUE THREAD

**Issue by Tal500** [sveltejs/kit#7353]
### Describe the problem

It is better to use [`aria-current`](aria-current="page") on the main page menu for accessibility, to specify what page is the current one (and the one that is the current, should be with `aria-current="page"`).

A good side effect is that we can use the `aria-current="page"` as a CSS selector to style the currently active page item.

### Describe the proposed solution

Use `aria-current`.

In Sapper demo, everything I have just said is implemented in the [old `Nav` component](https://github.com/sveltejs/sapper-template/blob/master/src/components/Nav.svelte), but why not in SvelteKit demos?

### Alternatives considered

Not doing this :-(

### Importance

nice to have
